### ISSUE TITLE
Settings model for a manually download model stopped working

### ISSUE THREAD

**Issue by ghost** [alexandersokol/sd-model-organizer#17]
Before the problem come, I downloaded models with my browser because it download more fast, after that I create the new model in model organizer and the field to set what existing model to use show the list of available models. 

Now that stopped working, I follow the same process, first I move my models in correct stable diffusion folder, I set if it's a checkpoint, a lora or an embedding and the field to select the model name not show anymore. What I suspect it's because I have downloaded these models before, deleted them entirely in model organizer but it possibly stay a data about the model that enter in conflict when I recreate a new one.

**Comment by alexandersokol** [comment]
Hi, I have to check it - the logic here should be kinda simple: If the record is new or it doesn't have file location binded - display this model name filed. Fills like it already has this file location field saved but isn't exists on disk. Are you able to check content of your `database.sqlite`? There should be a column called `location`.

PS: You confused me with download speed and I tried to download file with browser and speed was really faster. I send about an hour trying to understand how is it possible till I found out that my browser download folder is on SSD but model dir is on HDD...

**Comment by ema7569** [comment]
Yes more speed with firefox. A this hour, currently is around 40mb/s with firefox . With the model organizer interface it's around 7mb/s.  Few time, probably depending from server charge, firefox can do around 100mb/s, I'm on fiber, just ssd, no hdd.

Also, I downloaded https://sqlitebrowser.org/dl/ to open the sqlite database, say me if you need I test a thing. I have see the downloaded models can also have a file .info with tons of interesting things, but not all models have this .info file.

**Comment by ema7569** [comment]
Ha I progress in tests. I downloaded Realisation checkpoint with the interface that ok but in card view the option to downloaded it is here. Then I clicked download and start download and I have the error message:  Download failed. str: \'records\'

The realisian checkpoint is in the stable diffusion folder. In the database the realisian row the fields seems ok but missing the field download_filename is empty. I entered manually the download_filename in the db and now that work, the interface don't offer me to download anymore.

**Comment by alexandersokol** [comment]
I figured out that there might be an issue with filename string encoding for "Content-Disposition" header in the server responses. It is actually fixed in the main branch, and looks like that 'Realisian' model works fine. Is that issue still reproduces for you?
